packages/components/package.json

```diff
@@ -128,7 +128,7 @@
     "invariant": "2.2.2",
     "keycode": "2.2.0",
     "prop-types": "15.5.10",
-    "react": "^15.6.2 || ^16.0.0",
+    "react": "^16.0.0",
```

Comment:
The peer dependency here should include 15.
Plus, it doesn't fit the script, that introduce `|| 16`